src/Response/ZipResponse.php

```diff
@@ -82,7 +83,7 @@ public function getCsvResponse(): ?CsvResponse
             return new CsvResponse($this->responseParser, $csv);
         }
 
-        return null;
+        throw new InvalidZipResponseException('Zip Response doesn\'t contain the required CSV segment');
```

Comment:
Please let's have an exception code for this exception. I propose the current timestamp: 1567429445

Reply:
Update incoming…